documents/Specification/MaterialX.PBRSpec.md

```diff
@@ -374,6 +389,27 @@ Note that the standard library includes definitions for [**`displacement`**](./M
     * `ior` (**output**, vector3): Computed index of refraction.
     * `extinction` (**output**, vector3): Computed extinction coefficient.
 
+<a id="node-chiang-hair-roughness"> </a>
+
+* **`chiang_hair_roughness`**: Converts the artistic parameterization hair roughness to roughness for R, TT and TRT lobes. Output type `multioutput`, `roughness_R`, `roughness_TT` and `roughness_TRT`, `vector2` type. 
```

Comment:
might be good to explicitly mention Chiang paper for this node too
ie
`Converts the artistic parameterization hair roughness to roughness for R, TT and TRT lobes, as described in [^Chiang2016]`

Reply:
sounds good. added.